### PR TITLE
feat: custom overflow rule Modal component

### PR DESCRIPTION
**Pull request by dani69654**
Alt Text

### CODE REVIEW

**Comment by dani69654** [comment]
Added a `canOverflow` (bool) prop to add custom overflow rule if needed. <br />
Devs can now choose if the content of the modal should overflow (useful in case of dropdown menus)

File: src/components/Modal/Modal.styles.ts [hunk 1]
```diff
@@ -3,8 +3,24 @@ import colors from '../../styles/colors';
...
+        ? ''
+        : `
+                overflow: auto;
+                ::-webkit-scrollbar {
+                    display: none;
+                }
+                scrollbar-width: none;
+                -ms-overflow-style: none;
+            `;
+};
```
Comment (locothedev) [inline src/components/Modal/Modal.styles.ts:23]: console.log

maybe its better to return the css after an if statement so we dont return an empty string;

`if(!v) return css`

**Review by locothedev — APPROVED** [review]
check minor comment

**Review by locothedev — APPROVED** [review]
nooiccee

**Pull request merged**